tests/conftest.py

```diff
@@ -88,7 +68,14 @@ def integration_path():
         if build.env.fs_supports_symlink():
             os.symlink(self_source, self_dest)
         else:  # pragma: no cover
-            _copy_dir(self_source, self_dest, ignore=['.git', '.nox', '.integration-sources'])
+            os.makedirs(self_dest)
+            for target in ('pyproject.toml', 'setup.cfg', 'LICENSE', 'src'):
```

Comment:
Some documentation here would be helpful, stating why we need to single out these files, and not others.

Reply:
It was just Windows was having trouble copying the whole directory, so I am now copying the required files. For context, since it can be hard to track, this is the code to create the project folder for python-build in the integration tests. For the other projects we just download the tarball, but for python-build we copy the current source.